java/vector/src/main/java/org/apache/arrow/vector/compare/RangeEqualsVisitor.java

```diff
@@ -146,6 +147,11 @@ public Boolean visit(BaseVariableWidthVector left, Range range) {
     return compareBaseVariableWidthVectors(range);
   }
 
+  @Override
+  public Boolean visit(BaseLargeVariableWidthVector left, Range value) {
+    throw new UnsupportedOperationException();
```

Comment:
Is this meant to be implemented later? maybe add some comment and a message

Reply:
Sounds good. Added some comments here.